src/services/auth/asyncStorage.tsx

```
@@ -47,7 +47,7 @@ interface JwtPayload {
 
 export const getTokenData = async (): Promise<JwtPayload | null> => {
   const tokenResponse = await getToken();
-  if (tokenResponse?.token) {
+  if (tokenResponse && tokenResponse.token) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)